tests/utils/shippable/sanity.sh

```diff
@@ -26,4 +26,5 @@ ansible-test sanity --color -v --junit ${COVERAGE:+"$COVERAGE"} ${CHANGED:+"$CHA
     --docker --base-branch "${base_branch}" \
     --exclude plugins/module_utils/vendor/ \
     --exclude scripts/ \
+    --exclude tests/utils/ \
```

Comment:
Why do you exclude the folder if you just fixed all issues with the scripts?

Reply:
After I fixed the few errors, the sanity test started to complain that we didn't use `url_open` from the ansible module_utils.

So I thought we should ignore the test files direclty.

The other fixes are good to have.